src/main/java/io/pakland/mdas/githubstats/domain/model/UserReview.java

```diff
@@ -34,14 +34,8 @@ public class UserReview {
   )
   private List<Comment> comments = new ArrayList<>();
 
-  public void addReview(Comment comment) {
-    comments.add(comment);
-    comment.setUserReview(this);
-  }
-
-  public void removeReview(Comment comment) {
-    comments.remove(comment);
-    comment.setUserReview(null);
+  public int sumCommentLength() {
```

Comment:
What it is normally done: using primitive number types, or object number types?

Reply:
Good point, we could use objects as it would allow for example to make `List<Integers>` from them or allow `null` values, but in our models we have `int` types (e.g. `Comment.length` is a primitive `int`) so I used primitives as well